index.test.js

```diff
@@ -17,6 +18,12 @@ test('replaces selectors', () => {
     html,
     :root,
     a { }
+  }
+  @media (prefers-color-scheme:light) {
```

Comment:
Can you add the `light` tests to a separated test (maybe one extra test for both cases together)? 

Our tests already too long, it will simplify tests.

Reply:
Separated tests for the light scheme to individual tests and added one extra test for both schemes.